client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1431,6 +1431,8 @@
             },
             templateUrl: 'scripts/superdesk-authoring/views/send-item.html',
             link: function sendItemLink(scope, elem, attrs) {
+                var parentScope = scope.$parent.$parent;
```

Comment:
seems like it's only setting some css classes - you can use `superdeskFlags` service for that:
```
superdeskFlags.flags.fetching = !!item;
```
it will add fetching class to `#main-container`